### PR TITLE
chore: Update References

### PR DESCRIPTION
**Pull request by vipinvkmenon**
## Purpose

- Code of Conduct
- Security
- Reuse

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type

What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [x] Documentation content changes
- [ ] Other... Please describe:

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -106,20 +106,20 @@ jobs:
...
-          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
-      - uses: actions/download-artifact@v4
-        with:
-          name: coverage-report
-      - name: SonarCloud Scan
-        uses: SonarSource/sonarcloud-github-action@master
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  # sonarcloud:
```
Comment (vipinvkmenon) [inline .github/workflows/test.yml:109]: this is temporary

File: migration-guide/resources/app.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Provides a Cloud Foundry resource to manage applications.
 
-|  SAP Cloud Foundry Provider |Community Cloud Foundry Provider |
+| cloudfoundry/cloudfoundry |Community Cloud Foundry Provider |
```
Comment (Dray56) [inline migration-guide/resources/app.md:5]: Shouldn't we follow the same convention of replacing the Community Cloud Foundry Provider header with cloudfoundry-community/cloudfoundry as provided in the migration guide of data sources?

**Comment by Dray56** [comment]
The examples folder too includes references to sap/cloudfoundry registry which need to be changed on next release

File: README.md [hunk 3]
```diff
@@ -25,20 +24,20 @@ For the best experience using the Terraform Provider for Cloud Foundry, we recom
...
+Members, contributors, and leaders pledge to make participation in our community a harassment-free experience. By participating in this project, you agree to always abide by its [Code of Conduct](https://www.cloudfoundry.org/wp-content/uploads/2015/09/CFF_Code_of_Conduct.pdf).
 
 ## Licensing
 
-Copyright 2024 SAP SE or an SAP affiliate company and `terraform-provider-cloudfoundry` contributors. See our [LICENSE](LICENSE) for copyright and license information. Detailed information, including third-party components and their licensing/copyright information, is available [via the REUSE tool](https://api.reuse.software/info/github.com/SAP/terraform-provider-cloudfoundry).
+Copyright 2024 SAP SE or an SAP affiliate company and `terraform-provider-cloudfoundry` contributors. See our [LICENSE](LICENSE) for copyright and license information. Detailed information, including third-party components and their licensing/copyright information, is available [via the REUSE tool](https://api.reuse.software/info/github.com/cloudfoundry/terraform-provider-cloudfoundry).
```
Comment (KesavanKing) [inline README.md:43]: Do we require Copyright SAP SE ? Can we check if this Licensing is necessary ?

File: migration-guide/resources/app.md (hunk 2 above)
Comment (vipinvkmenon) [reply migration-guide/resources/app.md]: Good catch. Updated :)

File: README.md (hunk 3 above)
Comment (vipinvkmenon) [reply README.md:43]: This is accepted. Checked with Beyhan

Comment (vipinvkmenon) [reply README.md:43]: The ones to be changed...Code of Conducted & Security is updated

**Review by Dray56 — APPROVED** [review]

**Pull request merged**